ext/zlib/tests/gzgetss.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+gzgetss — Get line from gz-file pointer and strip HTML tags - function
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br>
+--SKIPIF--
+<?php
+if(!extension_loaded("zlib")){die("skip - ZLIB extension not loaded");}
+?>
+--FILE--
+<?php
+$handle = gzopen('gzgetss.gz', 'r');
+while (!gzeof($handle)){
+   $buffer = gzgetss($handle, 4096);
+   print($buffer);
+}
+gzclose($handle);
+?>
+--EXPECT--
+string gzgetss(resource $zp, int $length [, string $allowable_tags ]);
```

Comment:
I suggest to make the test more meaningful by using a small HTML file for gzgetss.gz; gzgetss() is supposed to strip out any tags.